### ISSUE TITLE
Class labels

### ISSUE THREAD

**Issue by bllfpc** [bllfpc/TTokDak#4]
- Class가 1개인 경우 
segmentation문제에서는 mask의 pixel값은 0 or 1이기 때문에 sigmoid로 할 수 있었지만 

- Class가 여려개인 경우 
mask의 pixel값은 0~m개가 될 수 있다.(m=1,2,3...) output layer에서 softmax로 7개의 units을 뽑아 낼건데, 그러면 각 픽셀별로 7개의 unit을 가지고 있을거라는 얘기고 그렇다면 마찬가지로 label도 각 pixel별로 7개의 units을 갖게 one-hot-vector로 치환해야하는건지 확인

**Comment by bllfpc** [comment]
1차원으로 flatten한 뒤에 sklearn의 preprocessing api로 label encoding으로 각 class 값을 변환해준뒤 one hot encoding을 한다.
